src/service/video-manager-impl.js

```diff
@@ -691,62 +821,53 @@ class VideoEntry {
    * @private
    */
   animateDocking_() {
-    // Minimize the video
-    this.video.hideControls();
-    this.internalElement_.classList.add(DOCK_CLASS);
+    // Calculate offsetXLeft
+    const offsetXLeft = this.calcDockOffsetXLeft();
```

Comment:
nit: most of these methods can become private, maybe do a sweep of public/private methods